Lumpen Radio/Radio.swift

```diff
@@ -56,32 +62,89 @@ class Radio: ObservableObject {
     // Reference:
     // https://stackoverflow.com/questions/48555049/how-to-stream-audio-from-url-without-downloading-mp3-file-on-device
     func startRadio() {
+        if self.player != nil {
+            self.player?.play()
+            self.isPlaying = true
+            self.bottomText = BOTTOM_TEXT_RADIO_ON
+            return
+        }
         guard let url = URL(string: LUMPEN_AUDIO_URL) else {
             print("Could not start radio")
             return
         }
         let playerItem = AVPlayerItem(url: url)
         let player = AVPlayer(playerItem: playerItem)
-        let audioSession = AVAudioSession.sharedInstance()
-        do {
-            try audioSession.setCategory(.playback,
-                                         mode: .default,
-                                         options: [.mixWithOthers, .allowAirPlay])
-            try audioSession.setActive(true)
-            player.play()
-            self.isPlaying = true
-            self.bottomText = BOTTOM_TEXT_RADIO_ON
-            self.player = player
-        } catch {
-            print(error)
-        }
+        
+        player.play()
+        self.isPlaying = true
+        self.bottomText = BOTTOM_TEXT_RADIO_ON
+        self.player = player
     }
```

Comment:
Forgot to add this feedback:

You can refactor this method a bit to clean it up by inverting the player nil check:

```
    func startRadio() {
        // Set up player if it's nil
        if self.player == nil {
            guard let url = URL(string: LUMPEN_AUDIO_URL) else {
                print("Could not start radio")
                return
            }
            let playerItem = AVPlayerItem(url: url)
            self.player = AVPlayer(playerItem: playerItem)
        }
        
        self.player?.play()
        self.isPlaying = true
        self.bottomText = BOTTOM_TEXT_RADIO_ON
    }
```

This way we avoid repeating code.